local-tests/tests/wrapped-keys/util.ts

```diff
@@ -17,6 +17,8 @@ export function getChainForNetwork(network: LIT_NETWORKS_KEYS): {
       };
     case 'datil-dev':
       return { chain: 'vesuvius', chainId: LIT_CHAINS['datilDevnet'].chainId };
+    case 'datil-test':
+      return { chain: 'vesuvius', chainId: LIT_CHAINS['datilTestnet'].chainId };
```

Comment:
Should both the `datil-dev` and `datil-test` case now use `chronicleVesuviusTestnet`?  I don't think there is an entry for `datilTestnet` in `LIT_CHAINS`
```suggestion
      return { chain: 'vesuvius', chainId: LIT_CHAINS['chronicleVesuviusTestnet'].chainId };
    case 'datil-test':
      return { chain: 'vesuvius', chainId: LIT_CHAINS['chronicleVesuviusTestnet'].chainId };
```